demisto_sdk/commands/unify/unifier.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, input: str, dir_name=INTEGRATIONS_DIR, output: str = '',
             # Also,
             # we don't take the unified file by default because
             # there might be packages that were not created by the plugin.
-            if 'unified' not in path:
+            if 'unified' not in path and os.path.basename(os.path.dirname(path)) not in [SCRIPTS_DIR, INTEGRATIONS_DIR]:
```

Comment:
what if I'm running the script on a package unrelated to content repo?
I don't think that we should make decisions based on file path in this flow.